src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -308,7 +332,18 @@ void FullyConnected::prepareParams() {
     NodeDesc *selected_pd = getSelectedPrimitiveDescriptor();
     if (selected_pd == nullptr)
         IE_THROW() << "Preferable primitive descriptor is not set for node " << getName() << ".";
-
+#ifdef OV_CPU_WITH_MLAS
+    // M, N should be normalized and updated
```

Comment:
What about N dimension? It is not updated below. Should we edit the comment?

Reply:
N dimension is fixed in this fc case, let me remove this